infra/tf/CONTRIBUTING.md

```diff
@@ -0,0 +1,5 @@
+# Contributing guidelines
```

Comment:
Is this for contributing to one of our terraform modules? Is this in the Engineering Playbook as a template for use in those repos?

Reply:
Nah, it's here because putting it in truss-infra would mean folks outside the org couldn't access it. After checking something in here, my plan was to add a link to the terraform repos back to these contributing guidelines. Do you have thoughts about a better single place to drop these guidelines?

Reply:
I did not realize the playbook was public. I think your plan is a good one, carry on.